### ISSUE TITLE
Stack accordion accessibility

### ISSUE THREAD

**Issue by chrysn** [Opener-Initiative/website#2]
The accordion of the stack lacks accessibility in that it is not using standard HTML elements, which (as I recently learned) do support this kind of expansion. Thanks to the latest round of @po3ts's refactoring, this is the only remaining part of the website that requires scripting.

I don't think this is urgent, and I'd leave this open to myself for half an hour of "I need to do something completely different from what I usually do".
